src/main/java/edu/cornell/kfs/rass/batch/RassValueConverterBase.java

```diff
@@ -20,39 +22,50 @@ public void setDataDictionaryService(DataDictionaryService dataDictionaryService
     }
 
     @Override
-    public Object convert(Class<? extends PersistableBusinessObject> businessObjectClass, String propertyName, Object propertyValue) {
-        return cleanSimplePropertyValue(businessObjectClass, propertyName, propertyValue);
+    public Object convert(Class<? extends PersistableBusinessObject> businessObjectClass, RassPropertyDefinition propertyMapping, Object propertyValue) {
+        return cleanSimplePropertyValue(businessObjectClass, propertyMapping, propertyValue);
     }
 
     protected Object cleanSimplePropertyValue(
-            Class<? extends PersistableBusinessObject> businessObjectClass, String propertyName, Object propertyValue) {
+            Class<? extends PersistableBusinessObject> businessObjectClass, RassPropertyDefinition propertyMapping, Object propertyValue) {
         if (propertyValue instanceof String) {
-            return cleanStringValue(businessObjectClass, propertyName, (String) propertyValue);
+            propertyValue = cleanStringValue(businessObjectClass, propertyMapping, (String) propertyValue);
```

Comment:
If feasible, you may want to avoid reassigning the method argument.

Reply:
I'll reoranize